app/code/Magento/Checkout/view/frontend/web/template/minicart/content.html

```diff
@@ -57,7 +58,8 @@
                             attr: {
                                 title: $t('Proceed to Checkout')
                             },
-                            click: closeMinicart()
+                            click: closeMinicart(),
```

Comment:
@rogerdz 
I think that changing click: closeMinicart() to click: closeMinicart would be enough here, `clickBubble` can be dropped